vitals/Vitals.java

```diff
@@ -0,0 +1,10 @@
+package vitals;
+
+public interface Vitals {
+
+    boolean checkVitalsRangeStatus();
+
+    static boolean checkRange(float value, float lowerLimit, float upperLimit){
```

Comment:
method name does not indicate that it returns a bool value. Naming can be more intuitive.